spf/dnscache.go

```diff
@@ -0,0 +1,26 @@
+package spf
+
+type dnsCache map[string]map[string][]string
```

Comment:
I don't see this as a general purpose dns cache as much as a utility for caching spf txt records.